bears/general/IndentationBear.py

```diff
@@ -379,6 +379,22 @@ def get_first_unindent(indent,
     return line_nr
 
 
+# TODO remove these once https://github.com/coala-analyzer/coala/issues/2377
+# gets a fix
+def lt_eq(absolute, source):
+    if absolute.line == source.line:
+        return absolute.column <= source.column
+
+    return absolute.line <= source.line
```

Comment:
I think you can just match `absolute.line < source.line`. You have already confirmed that they aren't equal in previous lines.